maestro/audit.py

```diff
@@ -56,6 +57,12 @@ def _format_what_compact(self, what):
         return '{} container{}'.format(
             len(what), 's' if len(what) > 1 else '')
 
+    def _format_what_services_ships(self, what):
+        if isinstance(what, entities.Entity):
```

Comment:
We reviewed this and i was trying to exclude just containers - wanted to work with the whole list, so I'm assuming this is ok